web/beacon-app/src/components/MenuDropdown/MenuDropdown.tsx

```diff
@@ -107,6 +106,7 @@ const MenuDropdownMenu = ({ items, isOpen, onOpenChange }: RadixDropdownMenuProp
                 </span>
               </DropdownMenuPrimitive.Item>
             )}
+            <DropdownMenuPrimitive.Arrow />
```

Comment:
I added the arrow after noticing one used on another site. I thought that it would help user's visually understand where the menu is coming from.